src/debate/debate.js

```diff
@@ -285,7 +295,23 @@ export class Debate {
         }
 
         let newLength = question.answers.push({answer: answer, uuid: socket.uuid});
-        this.clients[socket.uuid].answers[questionId] = newLength - 1;
+        let responseId = newLength - 1;
+        this.clients[socket.uuid].answers[questionId] = responseId;
+
+        //TODO: - Control if await slows down the app
+        //      - If it slows down the app, remove it and modify tests
+        //          (currently only pass with await otherwise they are executed too quickly)
+        await dbManager.saveResponse(responseId, answer, questionId, this.debateID, socket.uuid)
+            .then(res => {
+                if (res === true) {
+                    logger.info('Response saved to db');
+                } else {
+                    logger.warn('Response to db');
```

Comment:
The log message doesn't seems right. Isn't more like 'Failed to save response to db' ?

Reply:
Fixed it on f9dcfec. New message is ``logger.warn('Cannot save response to db');``